mail/forms.py

```diff
@@ -21,7 +21,11 @@ class MailTemplateUpdateForm(MailTemplateForm):
     def __init__(self, *args, **kwargs):
         super(MailTemplateForm, self).__init__(*args, **kwargs)
         if self.instance.department:
-            used = MailTemplate.objects.values_list('usage', flat=True).filter(department = kwargs["instance"].department)
-            used = [x for x in used if str(self.instance.usage) not in x]
+            used_db = MailTemplate.objects.values_list('usage', flat=True).filter(department = kwargs["instance"].department)
+            used = []
+            for x in used_db:
+                if x:
```

Comment:
always check for None with `if x is not None`

Reply:
why? I thought if x: did the same thing?

Reply:
ok, I forgot the false case